src/OrbitGl/OrbitApp.cpp

```diff
@@ -348,66 +348,68 @@ void OrbitApp::OnCaptureStarted(const orbit_grpc_protos::CaptureStarted& capture
+                                       std::move(frame_track_function_ids)]() mutable {
+    absl::flat_hash_map<Track::Type, bool> track_type_visibility;
+    bool had_capture = capture_window_->GetTimeGraph();
+    if (had_capture) {
+      track_type_visibility =
+          capture_window_->GetTimeGraph()->GetTrackManager()->GetAllTrackTypesVisibility();
+    }
 
-        ClearCapture();
+    ClearCapture();
 
-        // It is safe to do this write on the main thread, as the capture thread is suspended until
-        // this task is completely executed.
-        ConstructCaptureData(capture_started, file_path, std::move(frame_track_function_ids),
-                             data_source_);
-        GetMutableCaptureData().set_memory_warning_threshold_kb(
-            data_manager_->memory_warning_threshold_kb());
-        capture_window_->CreateTimeGraph(&GetMutableCaptureData());
-        orbit_gl::TrackManager* track_manager = GetMutableTimeGraph()->GetTrackManager();
-        track_manager->SetIsDataFromSavedCapture(data_source_ ==
-                                                 CaptureData::DataSource::kLoadedCapture);
-        if (had_capture) {
-          track_manager->RestoreAllTrackTypesVisibility(track_type_visibility);
-        }
+    // It is safe to do this write on the main thread, as the capture thread is suspended until
+    // this task is completely executed.
+    ConstructCaptureData(capture_started, file_path, std::move(frame_track_function_ids),
+                         data_source_);
+    GetMutableCaptureData().set_memory_warning_threshold_kb(
+        data_manager_->memory_warning_threshold_kb());
+    capture_window_->CreateTimeGraph(&GetMutableCaptureData());
+    orbit_gl::TrackManager* track_manager = GetMutableTimeGraph()->GetTrackManager();
+    track_manager->SetIsDataFromSavedCapture(data_source_ ==
+                                             CaptureData::DataSource::kLoadedCapture);
+    if (had_capture) {
+      track_manager->RestoreAllTrackTypesVisibility(track_type_visibility);
+    }
 
-        frame_track_online_processor_ =
-            orbit_gl::FrameTrackOnlineProcessor(GetCaptureData(), GetMutableTimeGraph());
+    frame_track_online_processor_ =
+        orbit_gl::FrameTrackOnlineProcessor(GetCaptureData(), GetMutableTimeGraph());
 
-        ORBIT_CHECK(capture_started_callback_ != nullptr);
-        capture_started_callback_(file_path);
+    ORBIT_CHECK(capture_started_callback_ != nullptr);
+    capture_started_callback_(file_path);
 
-        if (!GetCaptureData().GetAllProvidedScopeIds().empty()) {
-          ORBIT_CHECK(select_live_tab_callback_);
-          select_live_tab_callback_();
-        }
+    if (!GetCaptureData().GetAllProvidedScopeIds().empty()) {
+      ORBIT_CHECK(select_live_tab_callback_);
+      select_live_tab_callback_();
+      main_window_->SetLiveTabScopeStatsCollection(GetCaptureData().GetAllScopeStatsCollection());
```

Comment:
this seems to have triggered a reformatting of the entire lambda, but I wanted to highlight this line is the line that I actually changed.